src/VueTyping.spec.js

```diff
@@ -1,82 +1,90 @@
+function frame() {
+  return new Promise(resolve => window.requestAnimationFrame(resolve));
 }
 
-async function frames (numberOfFrames = 2) {
-    let currentFrame = 0
-    while (currentFrame < numberOfFrames) {
-        await frame()
-        currentFrame++
-    }
+async function frames(numberOfFrames = 2) {
+  let currentFrame = 0;
+  while (currentFrame < numberOfFrames) {
+    await frame();
+    currentFrame++;
+  }
 }
 
-describe('Basic rendering', () => {
-    let Vue, wrapper
-    beforeEach(() => {
-        Vue = createLocalVue()
-        wrapper = mount(VueTyping, {
-            propsData: {
-                text: 'Hello World!'
-            },
-            localVue: Vue
-        })
-    })
-
-    it('renders initial content', () => {
-        expect(wrapper.html()).toBe('<span>Hello World!</span>')
-    })
-    
-    it('Types one character per frame', async () => {
-        const text = 'Hey Guys, sup???'
-        wrapper.setProps({ text })
-        let currentText = ''
-        for (const char of text) {
-            await frame()
-            currentText += char
-            expect(wrapper.element.textContent).toBe(currentText)
+describe("Basic rendering", () => {
+  let Vue, wrapper;
+  beforeEach(() => {
+    Vue = createLocalVue();
+    wrapper = mount(VueTyping, {
+      propsData: {
+        text: "Hello World!",
+        cursorAnimationType: {
+          blinking: false,
+          cursor: "|",
+          color: "red",
+          framerate: 24
         }
-    })
+      },
+      localVue: Vue
+    });
+  });
 
-    it('Types one letter for each 2 frames when framerate is set to 2', async () => {
-        const text = 'Its me again, folks@!!'
-        wrapper.setProps({ text, framerate: 2 })
+  it("renders initial content", () => {
+    expect(wrapper.html()).toBe(
+      '<div><span>Hello World!</span><span style="opacity: 100; color: red;">|</span></div>'
+    );
+  });
 
-        // iterate over the array and count 2 frames each iteration
-        let currentText = ''
-        for (const char of text) {
-            await frames()
-            currentText += char
-            expect(wrapper.element.textContent).toBe(currentText)
-        }
-    })
+  it("Types one character per frame", async () => {
+    const text = "Hey Guys, sup???";
+    wrapper.setProps({ text });
+    let currentText = "";
+    for (const char of text) {
+      await frame();
+      currentText += char;
+      expect(wrapper.element.textContent).toBe(currentText + "|");
+    }
+  });
+
+  it("Types one letter for each 2 frames when framerate is set to 2", async () => {
+    const text = "Its me again, folks@!!";
+    wrapper.setProps({ text, framerate: 2 });
+
+    // iterate over the array and count 2 frames each iteration
+    let currentText = "";
+    for (const char of text) {
+      await frames();
+      currentText += char;
+      expect(wrapper.element.textContent).toBe(currentText + "|");
```

Comment:
These tests shouldn't test the blinking cursor, just the typing. It would be better to disable the cursor for them

Reply:
Alright, will do it.